docs/retired.rst

```diff
@@ -0,0 +1,68 @@
+#########################################
+Retired systems
+#########################################
+
+The following systems have been retired. For the full list of available systems, run `service.backends()`.  For more information, see  `Run on quantum backends <how_to/backends.html>`__.
+
+To retrieve results from retired backends, use `qiskit-ibm-provider <https://qiskit.org/documentation/partners/qiskit_ibm_provider/stubs/qiskit_ibm_provider.IBMBackend.html#qiskit_ibm_provider.IBMBackend>` or `QiskitRuntimeService.jobs <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.QiskitRuntimeService.jobs.html>`__.
+
+   
++-----------------------+----------------------+
+| System name           | Qubit count          |
++=======================+======================+
+| **Montreal**          | **27**               |
++-----------------------+----------------------+
+| **Toronto**           | **27**               |
```

Comment:
I don't think montreal and toronto are retired? I can still see them.

Reply:
They're retiring in March, according to this: https://github.ibm.com/IBM-Q-Software/sw-iqx-documentation/pull/1264

But I can pull them out for now and merge the change in its own branch later; depending on when the next Qiskit release is.  I'll follow up to find the actual date of retirement and the date of the next release.

Reply:
Looks like the next release is March 30, so I'll pull them out if the retirement isn't happening in the next couple of days.